src/views/tickets/Screenshot.js

```diff
@@ -6,54 +6,77 @@ import {
   Icon,
   Button,
   Modal,
+  Header,
 } from 'semantic-ui-react';
 import { getTicket, getArtifactURL } from '../../utils/api.js';
 
 import 'pure-react-carousel/dist/react-carousel.es.css';
 import { CarouselProvider, Slide, Slider } from 'pure-react-carousel';
 import CustomDotGroup from '../../components/CustomDotGroup.js';
 
+import { userAgentOptions } from '../../views/home/devices';
+const userAgents = userAgentOptions.devices;
+
 export default class Screenshot extends Component {
   state = {
     carouselLength: 1,
+    carousel: [],
   };
-  carousel = [];
-
   async getTicket() {
     // always get full page screenshot and add it to the carousel
     let image = (
-      <Slide index={0} width="100%">
+      <Slide index={0}>
+        <Header as="h4" dividing>
+          Default
+        </Header>
         <Image
           alt="fullpage screenshot"
           src={getArtifactURL(this.props.ticketID, 'screenshotFull.png')}
           bordered
           centered
+          fluid
         />
       </Slide>
     );
-    this.carousel.push(image);
+    let carouselTemp = [];
+    carouselTemp.push(image);
+    this.setState({ carousel: carouselTemp });
```

Comment:
if this is always one element, can inline: `this.setState({ carousel: [image] });`

Reply:
There's some weird stuff with setting states and arrays. When I try the code above it messes up the screenshots found in the carousel. I can look into it if you think that it would make the code better?

Reply:
nope, I believe you! react is super weird, and with babel it makes sense that some stuff works and others doesn't. still pretty confusing.